components/asset/introduction.rst

```diff
@@ -74,18 +75,27 @@ which defines the following two methods:
 :method:`Symfony\\Component\\Asset\\PackageInterface::getUrl`
     Returns an absolute or root-relative public path.
 
+With a package, you can:
+
+A) :ref:`version the assets <component-assets-versioning>`;
+B) set a :ref:`common base path (e.g. /css) <component-assets-path-package>`
+   for the assets;
+C) :ref:`configure a CDN <component-assets-cdn>` for the assets
```

Comment:
I LOVE ordered lists - they have the nice formatting of ordered but with little "reference" so we can refer to point "B" later if we want :)